src/Fantomas/FakeHelpers.fs

```diff
@@ -67,6 +65,13 @@ let formatFileAsync config (file : string) =
         | ex -> return Error(file, ex)
     }
 
+let formatFileAsync config (file : string) =
+    let originalContent = File.ReadAllText file
+    async {
+        return! originalContent
```

Comment:
Nitpick: I've never seen this construct before, maybe write it out the long way.

Reply:
Gotcha, I'll do it the long way around. 

`return!` just flattens an `Async<T>` result before exiting an `async` block, so you don't get an Async<Async<T>>.